roles/vdbench/templates/client.yaml

```diff
@@ -0,0 +1,67 @@
+---
+kind: Job
+apiVersion: batch/v1
+metadata:
+  name: 'vdbench-client-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+spec:
+  ttlSecondsAfterFinished: 600
+  template:
+    metadata:
+      labels:
+        app: vdbenchd-client-{{ trunc_uuid }}
+    spec:
+      containers:
+      - name: vdbench-client
+        image: "quay.io/avili/vdbench:latest"
```

Comment:
We will need to change this eventually (add it to SNAFU) Similar to how we build FIO - https://github.com/cloud-bulldozer/snafu/blob/master/fio_wrapper/Dockerfile

Reply:
You are right, this is my next step.

Reply:
i am waiting for snafu to be converted to python3

Reply:
@Avilir what I'd suggest is that you just add the dockerfile to snafu so that you don't have to maintain the image :)